proto/transport.proto

```diff
@@ -0,0 +1,288 @@
+      INACTIVE = 1;
+      ACTIVE = 2;
+    }
+
+    message Fault {
+      enum FaultType {
+        FAULT_UNSPECIFIED = 0;
+        CONTROLLER_FAULT = 1;
+        DRIVE_AND_MOTOR_FAULT = 2;
+        MECHANICAL_COMPONENT_FAULT = 3;
+        OVERSPEED_FAULT = 4;
+        POWER_SUPPLY_FAULT = 5;
+        SAFETY_DEVICE_FAULT = 6;
+        CONTROLLER_SUPPLY_FAULT = 7;
+        DRIVE_TEMPERATURE_EXCEEDED = 8;
+        // A comb plate safety switch is activated.
+        COMB_PLATE_FAULT = 9;
+        GENERAL_FAULT = 10;
+        DOOR_FAULT = 11;
+        LEVELLING_FAULT = 12;
+        SAFETY_CIRCUIT_BREAK_FAULT = 13;
+        FAIL_TO_START = 14;
+        // A fault has been detected with the alarm button. This is not the same as an activated passenger alarm.
+        // Use the passenger_alarm field to indicate the alarm button has been activated.
+        ALARM_BUTTON = 15;
+        DOOR_NOT_CLOSING = 16;
+        DOOR_NOT_OPENING = 17;
+        GOVERNOR_AND_SAFETY_GEAR_FAULT = 18;
+        // The fault is related to a device inside the lift shaft
+        LIFT_SHAFT_DEVICE_FAULT = 19;
+        CAR_STOPPED_OUTSIDE_LANDING_ZONE = 20;
+        CALL_BUTTON_STUCK = 21;
+        SELF_TEST_FAILURE = 22;
+        // The lift did not reach the expected zone.
+        RUNTIME_LIMIT_EXCEEDED = 23;
+        POSITION_LOST = 24;
+        // The car load measurement system is in a fault condition.
+        LOAD_MEASUREMENT_FAULT = 25;
+        OVERCAPACITY = 26;
+      }
+      // The type of fault.
+      FaultType fault_type = 1;
+      // Optional. The time the fault was detected.
+      google.protobuf.Timestamp time = 2;
+      // Optional. The description of the fault.
+      string description = 3;
+    }
+
+    // Journey describes the journey of the transport or payload.
+    // It will have a source and one or more destinations.
+    message Journey {
+      // The starting point of the journey.
+      Location start = 1;
+      // The destinations of the journey. These must be in chronological order of intended arrival time.
+      repeated Location destinations = 2;
+      // Optional. The reason for the journey.
+      string reason = 3;
+      // Optional. The time the journey started.
+      google.protobuf.Timestamp start_time = 4;
+      // Optional. The time the journey is estimated to arrive.
+      google.protobuf.Timestamp estimated_arrival_time = 5;
+    }
+
+    message Door {
+      enum DoorStatus {
+        DOOR_STATUS_UNSPECIFIED = 0;
+        CLOSED = 1;
+        OPEN = 2;
+        OPENING = 3;
+        CLOSING = 4;
+        // The door is safety locked.
+        SAFETY_LOCKED = 5;
+        // The door remains in a position that is not fully open or closed.
+        LIMITED_OPENED = 6;
+      }
+
+      // The id of the door, must be unique within the transport. This is used as the id of the door in door_status.
+      int32 id = 1;
+      // Description of the door; front, rear, etc.
+      string description = 2;
```

Comment:
Consider changing this to `display_name` or `title`. Description is more for things like `"Door used to access the upper floor"` whereas Title would be `"Primary"` or something like that.